drivers/sub8_videoray_m5_thruster/nodes/thruster_spin_test.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+
+if __name__ == '__main__':
+    import sys
```

Comment:
And everything for that matter.

This could be something that might be easier as an rqt plugin if Anthony is bent on making on